front/src/modules/trainschedule/components/Timetable/TimetableToolbar.tsx

```diff
@@ -140,100 +139,111 @@ const TimetableToolbar = ({
 
   return (
     <>
-      <div className="scenario-timetable-toolbar justify-content-between">
-        <div className="multi-select-buttons">
-          {multiSelectOn && (
-            <>
-              <input
-                type="checkbox"
-                className="mr-2"
-                checked={selectedTrainIds.length === trainSchedulesWithDetails.length}
+      <div
+        className={cx('scenario-timetable-toolbar justify-content-between', {
```

Comment:
Since we have a class anyway, maybe we could insert `justify-content` inside it ?

```suggestion
        className={cx('scenario-timetable-toolbar', {
```

Reply:
`justify-content-between` is not enabled in the browser, so I'll remove it.